render/marquee.go

```diff
@@ -44,6 +51,7 @@ type Marquee struct {
 	OffsetStart     int    `starlark:"offset_start"`
 	OffsetEnd       int    `starlark:"offset_end"`
 	ScrollDirection string `starlark:"scroll_direction"`
+	Align			string `starlark:"align"`
```

Comment:
no idea why the string is shifted here as it is not shown like that locally.....

Reply:
You can fix this (and a few other minor formatting issues) by running `go format` on the file.